nnvm/src/top/image/resize.h

```diff
@@ -0,0 +1,61 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file resize.h
+ */
+#ifndef NNVM_TOP_IMAGE_RESIZE_H_
+#define NNVM_TOP_IMAGE_RESIZE_H_
+
+#include <string>
+#include <vector>
+#include <utility>
+#include <iostream>
+#include <sstream>
+
+namespace nnvm {
+namespace top {
+
+template<typename ParamType>
+inline uint32_t UseResizeNumInputs(const NodeAttrs& attrs) {
```

Comment:
UseResizeNumInputs and UseUpsamplingNumInputs seem identical. Remove UseResizeNumInputs and call UseBilinearNumInputs.